caustic/simulator.py

```diff
@@ -17,4 +17,11 @@ class Simulator(Parametrized):
     """
 
     def __call__(self, *args, **kwargs):
-        return self.forward(self.pack(args[0]), *args[1:], **kwargs)
+        packed_args = self.pack(args[0])
+        return self.forward(packed_args, *args[1:], **kwargs)
+        # packed_args, batched = self.pack(args[0])
+        # out = self.forward(packed_args, *args[1:], **kwargs)
```

Comment:
why is all of this commented out? Is it no longer needed?

Reply:
Ah, forgot this here when testing ideas with batch dimension.